### DISCUSSION TITLE
After installation of the last update, Downloaded Video becomes Hazy

### DISCUSSION THREAD

**Discussion by jobair98** [aclap-dev/video-downloadhelper#359] [Help and Q&A]
### Your question

After installation of the last update, Video Capture is not clean and hazy. Please loot at the attached pic of screenshot. What Can I do? The Patch was on 31/3/2024. 
![Capture](https://github.com/aclap-dev/video-downloadhelper/assets/165726214/b422b742-6df1-40c5-b6b0-141c248e38cb)

### Addon & CoApp version and details

On the Date of 31 March, suddenly videodownloader asks me to install update, and thereafter I can download videos, but videos are hazy. No Sound, please see the attached videos pic.
![Capture](https://github.com/aclap-dev/video-downloadhelper/assets/165726214/d01bbf35-ec2f-47d5-963b-f96d74038565)

**Comment by Mattmjs** [comment]
DRM protection, can't do anything about that.
